vulnerable_people_form/templates/check-contact-details.html

```diff
@@ -7,6 +7,9 @@
   <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">
     {{title_text}}
     </h1>
+    {% if alt_email_suggestion %}
+    <p class="govuk-body">Did you mean <strong>{{ alt_email_suggestion }}</strong></p>
```

Comment:
should this have a ? at the end?

Reply:
source :

https://www.figma.com/file/yqy8aBWZsu5LuYnmIaGTj7/Extremely-vulnerable-people-form?node-id=2862%3A0

Reply:
yes it should